gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -1147,6 +1149,20 @@ private String toCase(String targetColumnName) {
     return columnName;
   }
 
+  /**
+   * Default DelimitedIdentifier should be 'double quotes',
+   * but that would make the column name case sensitive in case of Oracle.
+   * @return leftDelimitedIdentifier
+   */
+
+  public String getLeftDelimitedIdentifier() {
```

Comment:
Instead of making making the default delimiter empty, can you leave it as double quote, but make it optional for all databases? Users who need the escaping can set the parameter and we won't break existing configs.  Some databases like SqlServer have server-side config for case sensitivity and escaping lowercase names may not give desired behavior.